cmd/oci-create-runtime-bundle/main.go

```diff
@@ -81,6 +82,11 @@ func newBundleCmd(stdout, stderr *log.Logger) *cobra.Command {
 It is strongly recommended to keep the default value.`,
 	)
 
+	cmd.Flags().BoolVar(
+		&v.noSameOwner, "no-same-owner", false,
+		`Extract files as yourself instead of preserving the owner and group of each entry.`,
```

Comment:
The problem with having it `--same-owner` is that the default behaviour will be ignoring the ownership. The user will need to explicitly provide the flag, which sounds error-prone.

What do you mean by setting the flag on/off?